### ISSUE TITLE
Improve the Output Window on startup when commands are not shown

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#8091]
@ChrisMarsh82 I propose that the February release of R-Instat would change to not show the R commands.  

@Patowhiz if you agree with the suggestions below, then perhaps @N-thony could implement, or allocate.

(That would be less off-putting to some of our users, and also give us an extra "ease-of-use" point in the next round of evaluations of R GUIs!)

That introduces (at least) 3 points to be considered first:
**_1. What should the output window show on start-up?_**
@Patowhiz improved the look of the data window.  Here are the current windows - when commands are switched off:

![image](https://user-images.githubusercontent.com/11226469/215320948-13c0d1e7-98f9-4aaf-962b-4ff752131c51.png)

The output window doesn't look particularly inviting - or useful to me?

What should we put here.

I wonder about simply something like:
```

            Welcome to R-Instat
Data are shown in the Data View Window.
Results are shown in this Output Window.


```
But I am not imaginative on things like this?

**2. What should we show for the default comments.**

Here is the output window after opening a data frame and using the calculator a bit!

![image](https://user-images.githubusercontent.com/11226469/215320751-21c31f80-c0e1-492e-8039-dcfd84b03dc9.png)

The default comments are not useful - particularly when we are not showing the commands.  I still like the Comments "system" whereby a comment can be added with each dialogue.  

One possibility is to turn Comments off - in the Tools menu.  But I quite like the default of having them on, to encourage users to include useful comments.  Either way we could usefully change the default - which the user can do as well.

I have changed it to `Using the dialog:` - so also having a colon instead of the comma - as the dialogue name starts with a capital letter.

3. Should there be an additional simpler way of changing the default settings.  Perhaps even on the toolbar?

It could mirror the Tools menu, or be just within the current session if done that way?  If we don't want an extra icon, then perhaps  we just add a down arrow to the icon for the output window.  The first is then `Output Window` then `R Code` and `Comments`.  I suggest that's a good and neat place to provide this facility?

I quite like the idea that the change on the toolbar is for that session only.  If you want to change it permanently, then use the Tools  menu.

**Comment by rdstern** [comment]
@ChrisMarsh82 I believe the February update changed the default so that it now starts without showing commands?  If so, then  it would be good to consider some of the points above in the next versions?

**Comment by Patowhiz** [comment]
@rdstern I find comment "Code generated by the dialog" to not be useful when code is switched off. To me the key reason for the comment is the output not the code. I think users who are interested in the code will easily get used to using "To Script". Also, currently, the code and the output are shown as 2 separate outputs in the output window. And the comment currently explicitly aims at the code not the output itself. I was surprised why we had both options(switching off comments and code) separate. 

The broader questions are;

1. What's the aim of the comment? Is it to address the code or the output or both. 
2. Should we still have the comment and code as 2 separate outputs in the output window? This relates to the 2 switching options as well.
3. Should we add a second comment for the output itself? This one will always be associated with the output and therefore not independently selectable.


In regards to the default label when the code is switched off, would you support hiding the output window on start up? We could also display a different window instead, like the help. Then after a data set is opened, we just display it as usual.

**Comment by rdstern** [comment]
@Patowhiz interesting idea of yours to not have the Output Window, when we turn command off.  I tried it, and I think I prefer to keep our default of the 2 windows as they are, but with just a welcoming message - I have given my suggestion above.

Now the comments - I would like them to change anyway, so whether, or not we show commands.  It was Carlos, at the start of our development of R-Instat that suggested comments.  The idea is just like encouragingh a programmer to add explanatory comments to their code.  So I would like users to often replace the default, by an explanation of why they are doing that task, so what is their aim in producing the output.  Perhaps something like:  `The next step is the check whether a parallel-line model is adequate.`  or G`iven that there was no evidence of an interaction we jut give a one-way table of the means.`

So, I suggested above that the default comment could just change to `Using the dialogue: <name of dialogue as at the top> `as the default way af explaining the output.
